### DISCUSSION TITLE
github not find my js files

### DISCUSSION THREAD

**Discussion by albeeralkhawri** [community/community#77588] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

hi,

i have an issue with my new repo when i push the files to GitHub stop the JS and some image stop working when i track the issue see the browser unable to find the files the screenshot below

![image](https://github.com/community/community/assets/42253659/8de7dfec-9ca4-4f36-9ee5-6a7ce123ba95)
![image](https://github.com/community/community/assets/42253659/8253dcbb-8e2d-4793-9d7a-e4a6529ae058)
![image](https://github.com/community/community/assets/42253659/aa595199-cf03-4bca-bb5b-c45261797b8b)
![image](https://github.com/community/community/assets/42253659/c7dd91fb-a0f7-4f5f-a506-98bc940e350d)

please find the screenshot you will able to understand more about the issue 
https://albeeralkhawri.github.io/software-city/  this the demo link

please not the website working locally with no any issue 

Best Regards,
Albeer

**Comment by ColinHuelsbusch** [comment]
Hi, the files are not linked correctly. 

The images and JavaScript files are in the "software-city" subfolder not in the root folder.

https://albeeralkhawri.github.io/software-city/images/*.png not https://albeeralkhawri.github.io/images/*.png 
https://albeeralkhawri.github.io/software-city/js/*.js not https://albeeralkhawri.github.io/js/*.js 

You have to remove ../ before the images and ../../ before the JavaScript files.

Kind regards,
Colin

**Reply by albeeralkhawri** [reply]
Hi Colin

many thanks for your reply but i add them after the issue so they was linked directly to folder but the issue if you have a look on source the browser not load the js and images I can’t figure it why I think something in GitHub as they working in local with no issues 

Regards,
Albeer

**Reply by ColinHuelsbusch** [reply, marked as answer]
Hi Albeer,

if you add a slash in front of the path like /image it stats directly after the domain root and if you go up two directories like ../../js it will do the same. But in your case you need to start at /software-city/. Just change your code like this:

Replace 

```
  <script type="text/javascript" src="../../js/maps.js"></script>
  <script type="text/javascript" src="../../js/main.js"></script>
```

with

```
  <script type="text/javascript" src="js/maps.js"></script>
  <script type="text/javascript" src="js/main.js"></script>
```

and replace

```
  <span style="--i:5">
    <img src="/images/Cityofelectronics.png" alt="Citywebsite">
  </span>
  <span style="--i:6">
    <img src="/images/eletcreper.png" alt="eletwebsite">
  </span>
  <span style="--i:7">
    <img src="/images/group.png" alt="groupwebsite">
  </span>
  <span style="--i:8">
    <img src="/images/tv-menu.png" alt="menuwebsite">
  </span>
</div>
```

with 

```
  <span style="--i:5">
    <img src="images/Cityofelectronics.png" alt="Citywebsite">
  </span>
  <span style="--i:6">
    <img src="images/eletcreper.png" alt="eletwebsite">
  </span>
  <span style="--i:7">
    <img src="images/group.png" alt="groupwebsite">
  </span>
  <span style="--i:8">
    <img src="images/tv-menu.png" alt="menuwebsite">
  </span>
</div>
```

inside the index.html.

Kind regards,
Colin

**Comment by albeeralkhawri** [comment]
Hi colin

i will try and let you know

Regards,
Albeer

**Comment by albeeralkhawri** [comment]
Hi colin

it is working many thanks for your help

Best regards,
Albeer

**Reply by ColinHuelsbusch** [reply]
Hi Albeer,

I'm pleased to hear that. The website looks great!

I would greatly appreciate it if you could mark my post as answer.

Thanks,
Colin

**Reply by albeeralkhawri** [reply]
Sure i did many thanks i still work on it not finished it yet
